pilot/pkg/proxy/envoy/v2/cds.go

```diff
@@ -92,3 +99,68 @@ func (s *DiscoveryServer) generateRawClusters(node *model.Proxy, push *model.Pus
 	}
 	return rawClusters, nil
 }
+
+// Set the token path for SDS if SDS_TOKEN_PATH is defined in the proxy metadata
+func SetTokenPathForSdsFromProxyMetadata(c *xdsapi.Cluster, node *model.Proxy) {
```

Comment:
Please make it start with lowercase.

Reply:
The function name is uppercase because this function and its tests are in different packages.